README.md

```diff
@@ -1,5 +1,13 @@
 # Neo4j Graph Data Science Client
 
+[![Latest version](https://img.shields.io/pypi/v/graphdatascience)](https://pypi.org/project/graphdatascience/)
+[![PyPI downloads month](https://img.shields.io/pypi/dm/graphdatascience)](https://pypi.org/project/graphdatascience/)
+![Python versions](https://img.shields.io/pypi/pyversions/graphdatascience)
```

Comment:
Is there a `[` missing?

Reply:
added one :)

Reply:
No, that's only if you want to add a link to the badge. It's wrong now :see_no_evil: I will push a fix to main